CHANGELOG.md

```diff
@@ -5,6 +5,11 @@ Date format is Day-Month-Year
 
 ### Added
 
+###### 15-11-2018
+- Version 1.5.0 - Add STS authentication support
```

Comment:
If we are removing functionality then it is a breaking major change.
This should be 2.0.0